scripts/golinter.sh

```diff
@@ -13,10 +13,10 @@ declare -a vendoredModules=(
 "./integration/e2e"
 "./integration/fab3"
 "./integration/helpers"
-"./fab3"
 )
 
 declare -a goModules=(
+"./fab3"
 )
```

Comment:
Yes, I see it now, 2nd commit.

Reply:
yea not the cleanest division of the commits